### DISCUSSION TITLE
supporting RTL (right-to-left)

### DISCUSSION THREAD

**Discussion by arsalanyavari** [GitbookIO/community#76] [Feature Requests]
### What's your feature/idea?

Hello. The Arabic and Farsi language need to be RTL. They write and read from right to left, and the gitbook doesn't support RTL. The format of the text are being awful. Hamm it sounds to be good to add this feature to RTL LTR and other option for justify the text.
Thank you. 

### What's your desired outcome?

I have written a document in Farsi language for Linux, which helps to make it more readable. Also, in our country, to prepare a document, one of the virgol platforms is mainly used at the address https://virgool.io/ or https://telegra.ph/ which, according to Gitbook's good features, I think if such an option adding to it will increase the percentage of use of this platform and will help the teaching quality of university professors and TAs.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by software-ace** [comment]
I hope they do it as soon as possible 💙
